lib/src/models/movie.dart

```diff
@@ -16,10 +16,15 @@ class Movie {
   factory Movie.fromJson(
     Map<String, dynamic> parsedJson,
   ) {
-    var allMovies = parsedJson['results'] as List;
-
-    List<ResultMovie> movieTemp =
-        allMovies.map((eachMovie) => ResultMovie.fromJson(eachMovie)).toList();
+    List<ResultMovie> movieTemp;
+    if (parsedJson['results'] != null) {
+      var allMovies = parsedJson['results'] as List;
```

Comment:
Doing this without wrapping on a try catch can cause unwanted and unhanlded exceptions on the code

Reply:
Done! In case of the movies list callback were empty now its handled by a try/catch exception. Thank you Javi.